src/main/resources/resources/logisim/strings/std/std_fr.properties

```diff
@@ -768,7 +768,8 @@ TTL7411 = 7411 : triple 3 entrées ET porte d'entrée
 TTL74125 = 74125 : tampon quadruple bus, sorties à trois états, validation négative
 TTL74157 = 74157 : sélecteur de données quadruple de 2 lignes à 1 ligne
 TTL74158 = 74158 : quadruple 2-lignes to 1 ligne sélecteur, Sorties inversées
-TTL74139 = 74139 : quadruple 2-lignes to 4-lignes décodeur
+TTL74138 = 74138 : 3-lignes to 8-lignes décodeur
+TTL74139 = 74139 : double 2-lignes to 4-lignes décodeur
```

Comment:
"double 2-lignes to 4-lignes décodeur" -> "décodeur double de 2 lignes à 4 lignes"